docs/crashlytics/ios.md

```diff
@@ -4,8 +4,34 @@ description: Manually integrate Crashlytics into your iOS application.
 ---
 
 # iOS Setup
-
 > The following steps are only required if your environment does not have access to React Native
 auto-linking. 
 
-## TODO
+## Update Podfile
+
+```
+# Uncomment the next line to define a global platform for your project
+# platform :ios, '9.0'
+
+target 'MyProject' do
+# Comment the next line if you're not using Swift and don't want to use dynamic frameworks
+use_frameworks!
+
+# Pods for PodTest
+pod 'Fabric', '~> 1.10.1'
+pod 'Crashlytics', '~> 3.13.1'
```

Comment:
cc @ehesp - these should be using the versions config variables; not sure how they work though, can you help out here. Thanks

Reply:
```suggestion
pod 'Crashlytics', '~> {{ ios.firebase.crashlytics }}'
```

Is how it would go in v5.x.x but I think it will need a corresponding change in https://github.com/invertase/react-native-firebase/blob/master/docs/variables.yaml for it to work? It doesn't seem to have the majority of variables in there and I don't see corresponding files per package...